Compile failed on windows machine

* The crate version: `polling v2.5.0`
* The rustc version: `1.61.0`

The github action setup:
```yaml
    - if: matrix.os == 'windows-2019'
      name: Windows Dependencies
      run: |
        iwr -useb get.scoop.sh -outfile 'install-scoop.ps1'
        .\install-scoop.ps1 -RunAsAdmin
        echo "LIBCLANG_PATH=$($HOME)/scoop/apps/llvm/current/bin" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
        echo "$env:USERPROFILE\scoop\shims" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
        scoop install llvm yasm
```

The error message:
```
error[E0432]: unresolved import `std::ffi::c_int`
 --> C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\polling-2.5.0\src\wepoll.rs:4:5
  |
4 | use std::ffi::c_int;
  |     ^^^^^^^^^^^^^^^ no `c_int` in `ffi`

For more information about this error, try `rustc --explain E0432`.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -81,16 +81,19 @@ jobs:
         cargo build --target x86_64-unknown-illumos
 
   msrv:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
+        os: [ubuntu-latest, windows-latest]
         # When updating this, the reminder to update the minimum supported
         # Rust version in Cargo.toml.
         rust: ['1.47']
     steps:
       - uses: actions/checkout@v3
       - name: Install Rust
-        run: rustup update ${{ matrix.rust }} && rustup default ${{ matrix.rust }}
+        # --no-self-update is necessary because the windows environment cannot self-update rustup.exe.
+        run: rustup update ${{ matrix.rust }} --no-self-update && rustup default ${{ matrix.rust }}
       - run: cargo build
 
   clippy:

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+# Version 2.5.1
+
+- Fix the build error with MSRV on Windows
+
 # Version 2.5.0
 
 - Switch from `winapi` to `windows-sys` (#47)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,7 +3,7 @@ name = "polling"
 # When publishing a new version:
 # - Update CHANGELOG.md
 # - Create "v2.x.y" git tag
-version = "2.5.0"
+version = "2.5.1"
 authors = ["Stjepan Glavina <stjepang@gmail.com>"]
 edition = "2018"
 rust-version = "1.47"

--- a/src/wepoll.rs
+++ b/src/wepoll.rs
@@ -1,8 +1,8 @@
 //! Bindings to wepoll (Windows).
 
 use std::convert::TryInto;
-use std::ffi::c_int;
 use std::io;
+use std::os::raw::c_int;
 use std::os::windows::io::{AsRawHandle, RawHandle, RawSocket};
 use std::ptr;
 use std::sync::atomic::{AtomicBool, Ordering};

